tests/views/layout.jade

```diff
@@ -3,6 +3,7 @@ html
   head
     meta(charset='utf-8')
     link(type='text/css', rel='stylesheet', href='../node_modules/mocha/mocha.css')
+    link(rel='shortcut icon', href='data:image/x-icon;,', type='image/x-icon')
```

Comment:
Was just to get FF and Chrome to stop adding a warning to console about no favicon--but this didn't get added to `index.html` for some reason, despite my running the grunt test:local task...